File: PKMania/PM-DAL/Data/Entities/Tournament.cs

namespace PM_DAL.Data.Entities
{
    public class Tournament
    {
        public int Id { get; set; }
        public string Status { get; set; } = string.Empty;
        public DateTime StartedOn { get; set; }
        public DateTime FinishedOn { get; set; }
        public string Name { get; set; } = string.Empty;
        public int TournamentType { get; set; }
        public int RegistrationsNumber { get; set; }
        public int PrizePool { get; set; }
        public int RealPaidPlaces { get; set; }
        public int GainsSharingNr { get; set; }
        public IEnumerable<Gains>? Gains { get; set; }
        public int CurrentLevel { get; set; }
        public int CurrentSmallBlind { get; set; }
        public int CurrentBigBlind { get; set; }
        public int CurrentAnte { get; set; }
        public int NextLevel { get; set; }
        public int TimeBeforeNextLevel { get; set; }
        public int NextSmallBlind { get; set; }
        public int NextBigBlind { get; set; }
        public int NextAnte { get; set; }
    }
}

File: PKMania/PM-FrontEnd/src/app/features/tools/tournaments/models/tournament.ts
import { Gain } from "./gain";

export interface Tournament {
  id: number;
  status: string;
  startedOn: string;
  finishedOn: string;
  name: string;
  tournamentType: number;
  registrationsNumber: number;
  prizePool: number;
  realPaidPlaces: number;
  gainsSharingNr: number;
  gains: Gain[];
  currentLevel: number;
  currentSmallBlind: number;
  currentBigBlind: number;
  currentAnte: number;
  nextLevel: number;
  timeBeforeNextLevel: number;
  nextSmallBlind: number;
  nextBigBlind: number;
  nextAnte: number;
}
